examples/kubernetes-batch-demo/docker/Dockerfile.component-server

```diff
@@ -0,0 +1,34 @@
+# Dockerfile for Python Component Server
+# Provides HTTP-based component execution for Stepflow workflows
+
+FROM python:3.12-slim
+
+WORKDIR /app
+
+# Install uv for fast Python package management
+RUN pip install --no-cache-dir uv
+
+# Copy local stepflow-py SDK (build context is repo root)
+COPY sdks/python /tmp/stepflow-py
+
+# Install stepflow-py from local copy with HTTP support
+RUN uv pip install --system /tmp/stepflow-py[http]
```

Comment:
This should change back to the released stepflow server once we have it updated.

Reply:
#354